### DISCUSSION TITLE
WiX v5 Bootstrapper doesn't have ShowFilesInUse

### DISCUSSION THREAD

**Discussion by Bondarenko1990** [wixtoolset/issues#8698] [Questions]
Please tell me, in version 3 there was such a property ShowFilesInUse, but in version 5 there is no, or it is called differently. Tell me how to do this.

`<BootstrapperApplicationRef Id="WixStandardBootstrapperApplication.RtfLargeLicense">
      <bal:WixStandardBootstrapperApplication LicenseFile="..\MyApp\EULA.rtf" LogoFile="..\Graphics\InstallerLogo_64.png" 
                       ShowVersion="yes" **ShowFilesInUse="yes"** />
    </BootstrapperApplicationRef> `

**Comment by OlegSirenko** [comment]
I am pretty new to bootstrappers but in schema references I found this:
https://wixtoolset.org/docs/api/wixtoolset.mba.core/filesinusetype/

But I do not know why do you need `ShowFilesInUse="yes"`? For what purpose does this thing used in V3?

**Comment by Bondarenko1990** [comment]
@OlegSirenko I have a problem that when I uninstalling an application, a dialog appears FileInUse. I do not want this dialog to be displayed. That is, in Wix3 everything worked fine, in Wix 5 I get a dialog that should not be there.
[https://stackoverflow.com/questions/78896314/displayinternaluicondition-doesnt-work-correctly-wix-4](url)
